### ISSUE TITLE
Copy paste of 3rd "leg" of triplet does not work, copies previous contents of clipboard

### ISSUE THREAD

**Issue by sheetsofsound** [musescore/MuseScore#23492]
### Issue type

UI bug (incorrect info or interface appearance)

### Bug description

copy paste bug with triplets. I am trying to copy the voicing on the 3rd leg of a triplet. I use shift to highlight the 4 notes. When I paste, what gets pasted is something that was previously on the clipboard as you can see by the fact that it's pasting a chord which doesn't even exist at the point of the copy.

EDITORIAL COMMENT - YOU MAKE IT TOO HARD TO ADD ATTACHMENTS. Please support your native file format!!!!

And/or rar and other common formats...

### Steps to reproduce

create 1/4 note triplets as I have in the supplied chord. Copy the 3rd leg of one of the triplets and attempt to paste it into the first leg of another triplet. I use the menu in the video but normally I use ctrl-c / ctrl-v for the copy and the paste

### Screenshots/Screen recordings

https://github.com/musescore/MuseScore/assets/6884133/35e02b1d-2f17-4996-b20f-75685aded7cd

### MuseScore Version

OS: Windows 10 Version 2009 or later, Arch.: x86_64, MuseScore Studio version (64-bit): 4.3.2-241630831, revision: github-musescore-musescore-22b46f2

### Regression

I don't know

### Operating system

windows 11

### Additional context

[Diminished Dominants with compound triplet rhythms.zip](https://github.com/user-attachments/files/16116495/Diminished.Dominants.with.compound.triplet.rhythms.zip)

**Comment by MarcSabatella** [comment]
Copy/paste of partial tuplets is not supported.  There is supposed to be a dialog telling you this, bug I can confirm this dialog does not seem to appear in this case.  So, that would appear to be the bug here.

As a future enhancement, of course actually supporting partial tuplet copying in cases where it is possible *would* be nice.

BTW, unfortunately, the MuseScore team has no control over how GitHub handles attachments, but zipping files first is indeed the way to go.

**Comment by sheetsofsound** [comment]
OK, that's fine but it's more than an issue of not displaying a dialog box when you attempt to paste a partial tuplet. 

1. don't enable copy when you select a partial tuplet
2. don't enable paste when you select a partial tuplet
3. don't paste a completely different clipboard entry over top of a partial tuplet

**Comment by MarcSabatella** [comment]
Yes, the dialog is meant to do that - display the dialog and disable the operation, both the attempt to copy a partial tuplet and the attempt to paste into a tuplet.

**Comment by cbjeukendrup** [comment]
> YOU MAKE IT TOO HARD TO ADD ATTACHMENTS. Please support your native file format!!!!

This is obviously a limitation of GitHub, which we obviously have no control over.

**Comment by bkunda** [comment]
@zacjansheski pls check the issue about the dialog not appearing (which seems to be the primary issue here).
